pkg/systemd/reporting.jsx

```diff
@@ -226,6 +226,7 @@ class BusWorkflowRow extends React.Component {
                         this.setState({ message: _("Waiting for input…") });
                         const task_prompt = client.proxy("org.freedesktop.reportd.Task.Prompt", object_path);
                         const props = {
+                            title: _("Report to Fedora?"),
```

Comment:
Is this justified? Shouldn't "Fedora" come from some metadata of the backend?

Reply:
Oh good point! I think the text comes from reportd/taskd so this is :100: wrong.